pytest_rich.py

```diff
@@ -204,13 +207,14 @@ def _update_task(self, nodeid: str):
         completed = len(completed_count) == len(items)
         percent = len(completed_count) * 100 // len(items)
         description = f"[cyan][{percent:3d}%] [/cyan]{base_fn} " + "".join(chars)
-        self.runtest_progress.update(
-            task,
-            description=description,
-            refresh=True,
-            completed=completed,
-            visible=True,
-        )
+        if self.runtest_progress:
```

Comment:
In general I prefer an explicit check against None, as that `self.runtest_progress` object is not under our control and might change its `__bool__` method in the future, but really up to you. 👍

Reply:
Yeah, that's a good point. In my own applications, I reach for and prefer the simplicity of the implicit check, but it's purely an aesthetic choice. If it guards against any potential future bugs, I do not mind the explicit check.